CHANGELOG.md

```diff
@@ -7,6 +7,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.8.0] - 2022-12-20
+
+### Added
+- Added an environment variable `REMOVE_ACTIVE_FIELD`. When set, the `catalog_update`
+  script will remove the 'active' field for all versions of the given product.
+
+### Changed
+
```

Comment:
Delete extra line here.

Reply:
Hmm, didn't notice that before. Are you sure that I shouldn't actually have added blank lines under `### Added` and `### Fixed`? It looks like that is the pattern for older entries here in the change log.

Reply:
And for that matter, between the individual entries as well?

Reply:
Adjusted spacing in 1aa61e8b29f031626d62b5478cd8bffc9bcf14bf